docs/contributing/ways_to_contribute.rst

```diff
@@ -36,7 +34,7 @@ Please make sure to use the template. Replace the HTML comments (the ``<!-- ...
 Write code
 ----------
 
-See `How can I contribute? <https://github.com/learningequality/kolibri/blob/develop/CONTRIBUTING.md>`__
+Visit the `Contributing to our open code base <https://learningequality.org/contributing-to-our-open-code-base/>`__
```

Comment:
I don't think `the` is necessary here (or in the other two references) can just say Visit <page title>.